src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -748,6 +748,11 @@ private function resolveServices($value, $file, $isParameter = false)
 
                 return new Reference($id);
             }
+            if ('yaml_file' === $value->getTag()) {
+                $file = $this->container->resolveEnvPlaceholders($argument, true);
```

Comment:
We should define a root for relative files. I suggest using the file where the tag is found when it makes sense (when parsing a file), and to throw an exception otherwise.

Reply:
Also I don't think this should resolve env vars. Env vars are for runtime parameters, loading yaml is compile time.

Reply:
I am not sure if this makes sense to use it as root where the file is found, but maybe relative from project dir?

Reply:
The component doesn't know what a project for is.

Reply:
True

Reply:
@nicolas-grekas should we allow absolute path as well?

Reply:
Fixed. Tests are failing on Travis for PHP 7.3, but running on Docker it's working for me. Any idea what's going on?